slomonitor/src/main/scala/com.gu.notifications.slos/SloMonitor.scala

```diff
@@ -0,0 +1,68 @@
+package com.gu.notifications.slos
```

Comment:
This file contains the new Lambda code; it's probably the most interesting part to review.